mlrun/api/crud/model_endpoints.py

```diff
@@ -120,7 +137,83 @@ def create_or_patch(
 
         logger.info("Model endpoint updated", endpoint_id=model_endpoint.metadata.uid)
 
-        return model_endpoint
+    def create_monitoring_feature_set(
+        self,
+        model_endpoint: mlrun.api.schemas.ModelEndpoint,
+        model_obj: mlrun.artifacts.ModelArtifact,
+        db_session: sqlalchemy.orm.Session,
+    ):
+        """
+        Create monitoring feature set with the relevant parquet target.
+
+        :param model_endpoint:    An object representing the model endpoint
+        :param model_obj:         An object representing the deployed model
+        :param db_session:        A session that manages the current dialog with the database
+
+        """
+
+        # define a new feature set
+        serving_function_name = model_endpoint.spec.function_uri.replace(
+            model_endpoint.metadata.project + "/", ""
+        )
+        model_name = model_endpoint.spec.model.replace(":", "-")
+
+        feature_set = mlrun.feature_store.FeatureSet(
+            f"monitoring-{serving_function_name}-{model_name}",
+            entities=["endpoint_id"],
+            timestamp_key="timestamp",
+            description=f"Monitoring feature set for endpoint: {model_endpoint.spec.model}",
+        )
+        feature_set.metadata.project = model_endpoint.metadata.project
```

Comment:
i suggest to add `labels` w relevant info e.g. `feature_set.metadata.label["endpoint"] = endpoint_id`